pallets/btc-registration-pool/src/pallet/impls.rs

```diff
@@ -173,15 +173,15 @@ impl<T: Config> Pallet<T> {
 		signature: &T::Signature,
 		tag_prefix: &'static str,
 	) -> TransactionValidity {
-		let VaultKeySubmission { authority_id, who, pub_key } = key_submission;
+		let VaultKeySubmission { authority_id, who, pub_key, pool_round } = key_submission;
 
 		// verify if the authority is a relay executive member.
 		if !T::Executives::contains(authority_id) {
 			return Err(InvalidTransaction::BadSigner.into());
 		}
 
 		// verify if the signature was originated from the authority.
-		let message = array_bytes::bytes2hex("0x", pub_key);
+		let message = format!("{}:{:?}", pool_round, array_bytes::bytes2hex("0x", pub_key));
```

Comment:
```suggestion
		let message = format!("{}:{}", pool_round, array_bytes::bytes2hex("0x", pub_key));
```
양끝에 '"'가 붙습니다. 이하 동일